pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/model/patch/ParticipantDataPatch.java

```diff
@@ -52,7 +52,7 @@ Optional<Object> processEachNameValue(NameValue nameValue) {
         ParticipantDataNameValue participantDataNameValue =
                 new ParticipantDataNameValue(nameValue.getName(), nameValue.getValue(), ddpInstance.getDdpInstanceIdAsInt(),
                         patch.getDdpParticipantId(), patch.getFieldId());
-        exportToESWithId(participantDataId, participantDataNameValue);
+        //exportToESWithId(participantDataId, participantDataNameValue);
```

Comment:
shall we remove that, when it is now done per BasePatch

Reply:
good catch, totally forgot those comments.